pkg/release/release.go

```diff
@@ -38,13 +37,14 @@ type Config struct {
 
 type Release struct {
 	apprClient *apprclient.Client
-	fileLogger *filelogger.FileLogger
 	helmClient *helmclient.Client
 	k8sClient  kubernetes.Interface
 	logger     micrologger.Logger
 
-	condition *conditionSet
 	namespace string
+
+	condition  *conditionSet
```

Comment:
Why do we need the third block now?

Reply:
Internals.

Reply:
The order should be `deps`, `internals`, `settings` then according to fmt https://github.com/giantswarm/fmt/blob/master/go/struct_initialization.md

Reply:
It makes it harder to see what is passed from the config. But fine...